plugins/InspectionGadgets/InspectionGadgetsAnalysis/resources/messages/InspectionGadgetsBundle.properties

```diff
@@ -2206,6 +2206,7 @@ inspection.redundant.string.operation.display.name=Redundant String operation
 inspection.redundant.string.remove.fix.name=Remove redundant ''{0}()'' call
 inspection.redundant.string.fix.family.name=Remove redundant call
 inspection.redundant.string.call.message=Call to <code>#ref</code> is redundant #loc
+inspection.redundant.string.call.may.be.message=Call to <code>#ref</code> may be is redundant #loc
```

Comment:
"may be is redundant" sounds grammatically incorrect. "is probably redundant" would be better. In general, I don't see the necessity to create a separate message. To me, inspection.redundant.string.call.message is just fine.

Reply:
thank you for checking! I changed to "inspection.redundant.string.call.message"